examples/citi-bike/rides.config.yaml

```diff
@@ -0,0 +1,23 @@
+# The CitiBike tripdata bucket is public, and we don't require credentials.
+awsAccessKeyId: ""
+awsSecretAccessKey: ""
+# Name of the bucket managed by CitiBike.
+bucket: tripdata
+# We encrypt the region just to demonstrate the capability (this is "us-east-1").
+# We used a `sops` --encrypted-suffix "-sops" option to selectively encrypt only
+# some values of this configuration. Flow knows to strip this suffix.
+region-sops: ENC[AES256_GCM,data:OnrKzkp1aQGz,iv:NmUy9yeRdH1jvzVFuf/sKzMqAf6hEvOA5ohAsJ50O68=,tag:hUSkjQejD7cwzd4aPGlC1A==,type:str]
+sops:
+    kms: []
+    gcp_kms:
```

Comment:
This might prevent the citi-bike demo working on local machines without GCP settings.

I know this is just the start of `sops` integration, and before the work is completed, is it possible to keep a non-encrypted version of `rides.config.yaml` here, and override it with a encrypted version during integration tests, so that the demo can work for the needs from both local and testing environment?

Reply:
Yea. Today, `examples/` are doing double duty, as both examples and also integration tests.

There's a (very recently added) end-to-end test which is also run by CI. Today it's directly using this citi-bike example, and using sops here provides continuous coverage of that decryption.

But I'm not terribly happy with the end-to-end test in its current state (it's... okay), and would like to eventually revert this as part of improving that.

I'll note also that there are some other decryption tests that do require access to this same key. Not sure what to do about that, assuming we want CI coverage of `sops` usage 🤷